tests/NCronJob.Tests/NCronJobIntegrationTests.cs

```diff
@@ -204,22 +204,6 @@ public async Task LongRunningJobShouldNotBlockScheduler()
         jobFinished.ShouldBeTrue();
     }
 
-    [Fact]
-    public async Task ThrowIfJobWithDependenciesIsNotRegistered()
-    {
-        ServiceCollection
-            .AddNCronJob(n => n.AddJob<JobWithDependency>(p => p.WithCronExpression("* * * * *")));
```

Comment:
Do we still need the JobWithDependency type?

Reply:
No - I will remove the clutter.